test/cli_tests/test_demo.py

```diff
@@ -0,0 +1,28 @@
+import unittest
```

Comment:
there are snazzier test frameworks, but the built-in one, (originally based on junit), does fairly well.

this module is mostly meant as a demo/stand-in for actual tests. but at least it shows one way of doing rather functional-level tests of the CLI.

Reply:
I think built-in is the correct choice here, unless there is a real reason to move to something heavier